reference_points:
  apple:
    point: 
      x: -1.34327852726
      y: 2.03266811371
      z: 0.77642083168
  lemon:
    point: 
      x: -1.3492307663
      y: 1.75750803947
      z: 0.773347973824
  pear:
    point: 
      x: -1.33993434906
      y: 2.35632419586
      z: 0.76607131958
  toilet_paper:
    point: 
      x: -1.65527117252
      y: 2.02992939949
      z: 0.780992150307
